third-party/linshare/src/test/java/org/apache/james/linshare/client/TechnicalAccountCreationRequest.java

```diff
@@ -57,7 +58,7 @@ public String getMail() {
         return mail;
     }
 
-    public String getEnable() {
+    public boolean isEnable() {
```

Comment:
s/isEnable/isEnabled ?

Reply:
In this case, the returned file is `enable` and i used it but we can choose a better name for it.
IMO: the field `enable` is should be name `status` is better.

Reply:
Here it's more about english grammar : `is + adjective`

enable is a verb, hence the proposal of remi to use the `enabled` adjective